scanner/src/main/kotlin/storages/FileBasedStorage.kt

```diff
@@ -47,6 +47,8 @@ class FileBasedStorage(
      */
     private val backend: FileStorage
 ) : ScanResultsStorage() {
+    override val name = backend.javaClass.simpleName
```

Comment:
Maybe `"${javaClass.simpleName} with ${backend.javaClass.simpleName} backend"`?

Reply:
I was briefly considering that, but in most cases that would read something like "FileBasedStorage with LocalFileStorage backend", which seemed a but verbose to me regarding the multiple mentions of "file" and "storage". But I'm not totally opposed to the change... I'll think about it a bit more.

Reply:
Ok, done.